app/src/main/java/com/skyyo/samples/features/verticalPagerWithFling/VerticalPagerWithFlingViewModel.kt

```diff
@@ -15,9 +15,9 @@ class VerticalPagerWithFlingViewModel @Inject constructor() : ViewModel() {
         populateListWithFakeData()
     }
 
-    @Suppress("LongMethod")
+    @Suppress("LongMethod", "MagicNumber")
     private fun populateListWithFakeData() {
-        val testVideos = listOf(
+        listOf(
```

Comment:
when we are warned that variable isn't used here - just comment it out, don't delete nor create list that isn't used anywhere